bulbs/utils/methods.py

```diff
@@ -99,3 +100,12 @@ def get_overridable_template_name(parent_name, child_name):
         template_name = parent_name
 
     return template_name
+
+
+def get_video_object(videohub_id):
```

Comment:
I'd recommend a more explicit function name like `get_video_from_videohub_id`, in case we add similar helper methods in the future